tests/sqlalchemy_factory/test_sqlalchemy_factory_common.py

```diff
@@ -328,12 +330,18 @@ class Factory(SQLAlchemyFactory[AsyncModel]):
             __model__ = AsyncModel
```

Comment:
Factories by default don't use default values of the model. There is a setting but not sure if it works with SQLA factory properly. I think adding `test_datetime = Ignore()` to the factory would improve the test.

Reply:
I added a new test case for server_default func in async_session to test with other cases.